hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/disk/OnDiskValueSerializer.java

```diff
@@ -17,25 +17,28 @@
 package com.hedera.node.app.state.merkle.disk;
 
 import com.hedera.node.app.state.merkle.StateMetadata;
-import com.swirlds.common.io.streams.SerializableDataInputStream;
-import com.swirlds.common.io.streams.SerializableDataOutputStream;
-import com.swirlds.jasperdb.SelfSerializableSupplier;
+import com.swirlds.merkledb.serialize.ValueSerializer;
 import edu.umd.cs.findbugs.annotations.NonNull;
 import java.io.IOException;
+import java.nio.ByteBuffer;
 import java.util.Objects;
 
 /**
- * An implementation of {@link SelfSerializableSupplier}, required by the {@link
+ * An implementation of {@link ValueSerializer}, required by the {@link
  * com.swirlds.virtualmap.VirtualMap} for creating new {@link OnDiskValue}s.
  *
  * @param <V> The type of the value in the virtual map
  */
-public final class OnDiskValueSerializer<V> implements SelfSerializableSupplier<OnDiskValue<V>> {
+public final class OnDiskValueSerializer<V> implements ValueSerializer<OnDiskValue<V>> {
     @Deprecated(forRemoval = true)
```

Comment:
There is no value to annotating a variable as deprecated, particularly a private one.  Can we remove this?

Reply:
Sure, I'll remove it.